src/main/java/core/basesyntax/PurchaseOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class PurchaseOperation implements OperationHandler {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove all redundant empty lines to adhere to the coding style described in the checklist.